### ISSUE TITLE
Error When importing

### ISSUE THREAD

**Issue by wcastand** [expo/react-native-action-sheet#63]
Version Expo: 23.0.4
my package.json dependencies : 
```
    "@expo/react-native-action-sheet": "0.4.0",
    "@expo/vector-icons": "6.2.1",
    "date-fns": "2.0.0-alpha.7",
    "expo": "^23.0.4",
    "fela": "6.1.0",
    "fela-native": "5.0.14",
    "fela-plugin-extend": "6.0.2",
    "react": "16.0.0",
    "react-fela": "6.1.1",
    "react-native": "0.50.3",
    "react-native-animatable": "1.2.4",
    "react-navigation": "1.0.0-beta.21"
```

I tried to use action sheet and get an error when i import the package.
this is the piece of code : 

```javascript
// @flow
import React from "react";
import Expo from "expo";
import { AppRegistry } from "react-native";
import { Provider, ThemeProvider } from "react-fela";
import { Font } from "expo";
import { ActionSheetProvider } from "@expo/react-native-action-sheet";

import Root from "./root";
import renderer from "./fela";
import theme from "./theme";

class App extends React.Component {
  state = { fontLoaded: false };
  async componentDidMount() {
    await Font.loadAsync({
      "hk-grotesk-regular": require("./assets/fonts/HKGroteskRegular.ttf"),
      "hk-grotesk-bold": require("./assets/fonts/HKGroteskBold.ttf")
    });
    this.setState(() => ({ fontLoaded: true }));
  }
  render() {
    return this.state.fontLoaded ? (
      <Provider renderer={renderer}>
        <ThemeProvider theme={theme}>
          <Root />
        </ThemeProvider>
      </Provider>
    ) : null;
  }
}

export default App;
```

![img_0326](https://user-images.githubusercontent.com/2178244/33612582-5234cf96-d9d2-11e7-9fe9-f137a6123557.PNG)

If i can help or you need more info, let me know :)

**Comment by ER1011** [comment]
If you are using React 16 then this is likely a problem related to the PropTypes, which in react 16 have to be imported separately, like this:  import PropTypes from 'prop-types'

**Comment by wcastand** [comment]
I don't have any PropTypes in my app and i get the error msg when i add the import of action sheet.
`import { ActionSheetProvider } from "@expo/react-native-action-sheet";`

But action-sheet import PropTypes from `prop-types` and not react. 

So i don't see why i get this error.

**Comment by ER1011** [comment]
You may want to check in the .js files in the @expo/react-native-action-sheet on your machine. When I did the npm install of the repo yesterday, what came down was not the latest version of the code and the PropTypes were the issue for me as well

**Comment by wcastand** [comment]
You are right, the latest version on npm is not the last version.

@0.4.0 still uses PropTypes from 'react'.

**Comment by maxushuang** [comment]
#64 
Can you help with this issue? @ER1011 @wcastand

**Comment by wcastand** [comment]
The latest version on npm export an old version that still use PropType from React.
Any way we can help to push the new version to npm ?
https://unpkg.com/@expo/react-native-action-sheet@0.4.0/

**Comment by peterpme** [comment]
:wave: 

@ggepenyan 
@townmulti 
@wcastand 
@maxushuang 

This should be fixed with v1.0.2:
https://www.npmjs.com/package/@expo/react-native-action-sheet
